features/people_page_filter_check.feature

```diff
@@ -1,14 +0,0 @@
-Feature: Checking filter on the people page
```

Comment:
Is there a reason to remove this feature?

Reply:
added up